docs/modules/ROOT/pages/documentation/microprofile/faulttolerance.adoc

```diff
@@ -79,15 +79,15 @@ image:microprofile/fault-tolerance.png[Set Fault Tolerance Configuration]
 [[using-asadmin-commands]]
 === Using Asadmin Commands
 
-NOTE: Since Payara Platorm 5.2020.2, the configuration no longer allows configuring user defined executor to run the fault tolerance execution. Instead, Fault Tolerance uses two dedicated executors that are well suited for the usage pattern typically experienced when executing Fault Tolerance semantics. It's possible to configure the maximum size of these pools to scale them up or down.
+NOTE: On Payara Platform 5.2021.1, it is a known issue that the `set-fault-tolerance-configuration` command is broken and it can't be used to configure user-defined executor to run the fault tolerance execution. To work around this issue, use the following `set` command to configure managed executor pool: `asadmin set configs.config.server-config.microprofile-fault-tolerance-configuration.managed-executor-service` and similar use the following  command to configure scheduled managed executor pool: `asadmin set configs.config.server-config.microprofile-fault-tolerance-configuration..managed-scheduled-executor-service`. Finally, a server restart is required for the commands to take effect.
```

Comment:
Should we mention here already that a default executor is used an no longer a dedicated executor?